documentation/config_file_auth.md

```diff
@@ -0,0 +1,151 @@
+---
+layout: default
+title: "Puppet Server Configuration Files: auth.conf"
+canonical: "/puppetserver/latest/config_file_auth.html"
+---
+
+[`trapperkeeper-authorization`]: https://github.com/puppetlabs/trapperkeeper-authorization
```

Comment:
Have you thought any further about how we want to cover / link to the complete set of settings in the `trapperkeeper-authorization` "auth.conf" file format?  This document covers a subset of those - most of the interesting ones that users converting from the deprecated format to the new format would care about.  For more complete information - including information on the new required settings like `sort-order` and `name` that they'll need to implement - how are we thinking we'll direct users?  I'm definitely onboard with the idea of not linking directly to the "authorization-config.md" doc in the tk-authz github repo.  Do we want to eventually publish that doc to somewhere on docs.puppetlabs.com and link to that from this document?  Open to however you and @nfagerlund think would be best to do this...